charts/portworx-essentials/v1.0.0/README.md

```diff
@@ -1,9 +1,30 @@
-# Portworx
+# Portworx Essentials
+[Portworx Essentials](https://docs.portworx.com/concepts/portworx-essentials/) is a free Portworx license with limited functionality that allows you to run small production or proof-of-concept workloads. Essentials limits capacity and advanced features, but otherwise functions the same way as the fully-featured PX-Enterprise version of Portworx.
+
+The Portworx Essentials license requires that your clusters be connected to the internet and send usage data to PX-Central. Portworx Essentials clusters connect with PX-Central once per hour to renew license leases. Lease periods last for 24 hours, ensuring that any temporary interruptions to your connectivity do not impact your cluster.
 
 ## **Pre-requisites**
 
-Use this Helm chart to deploy [Portworx](https://portworx.com/) and [Stork](https://docs.portworx.com/scheduler/kubernetes/stork.html) to your Kubernetes cluster.
+The minimum supported size for a Portworx cluster is three nodes. Each node must meet the following hardware, software, and network requirements:
+
+### Hardware & Software
+	
+|Resource|Requirements|
+|--------|-----------|
+|CPU|4 cores|
+|RAM|4GB|
+|Disk (/var)| 2GB free|
+|Backing drive|8GB (minimum required) 128 GB (minimum recommended)|
+|Storage drives| Storage drives must be unmounted block storage: raw disks, drive partitions, LVM, or cloud block storage.|
```

Comment:
Add a note here that at-least one node in the cluster needs to have these storage drives

Reply:
added a minimum section.